apps/docs/content/guides/storage/s3/authentication.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: 'storage-s3-authentication'
+title: 'S3 Authentication'
+description: 'Authentication'
+subtitle: 'Learn about authenticating with Supabase Storage S3.'
+sidebar_label: 'S3'
+---
+
+You have two options to authenticate with Supabase Storage S3:
+
+- Using the generated S3 credentials from your project settings (Intended exclusively for server-side use)
+- Using a Session Token, which will allow you to authenticate with a user JWT token and provide limited access via RLS
+
+## S3 Credentials
+<Admonition type="note">
+Keep these credentials secure, they will provide full access to all S3 operations across all buckets (Bypassing any RLS policies)
+In a future release we will provide a way to limit the access of these credentials using RLS.
+</Admonition>
+
+To authenticate with S3 on the server, you need to generate a pair of credentials (Access Key ID and Secret Access Key) from your [project settings](https://supabase.com/dashboard/project/_/settings/storage).
+
+You can use these credentials to authenticate with Supabase Storage S3 as any other S3-compatible service.
+
+<img alt="Storage S3 Credentials" src="/docs/img/storage/s3-credentials.png" width="100%" />
+
+
+**Region**: The region is the same as the region you selected during project creation.
+**Endpoint**: `https://project_ref.supabase.co/storage/v1/s3`
+
+
+Example using the `aws-sdk` library:
+
+```javascript
+import { S3Client } from '@aws-sdk/client-s3';
+
+const client = new S3Client({
+  forcePathStyle: true,
+  region: 'project-region',
+  endpoint: 'https://project_ref.supabase.co/storage/v1/s3',
+  credentials: {
+    accessKeyId: 'your-access-key-id',
+    accessSecretKey: 'your-secret-access-key',
+  }
+})
+```
+
+## Session Token
+
+When using the Session Token mechanism, you will be able to authenticate with a user JWT token and provide limited access via RLS.
+This is very powerful because you can initialize the S3Client on the server scoped to a specific user or use the S3 client directly from the client-side without the need for a server.
+
+All S3 operations will be performed scoped by the authenticated user provided in `session_token` it will be subject to any RLS policies that you have applied to the Storage Schema.
+
+To authenticate with S3 using a Session Token, you will always use the following credentials:
+
+- access_key_id: `project-ref`
```

Comment:
Is this the token you get from `(await getSession()).data.access_token`? Perhaps it would be good to provide an example of how to get the user's token